File: conexiones/conexionPost.js
const { Client } = require('pg');

const client = new Client({
    host: 'localhost',
    user: 'postgres',
    password: '12345'
  //, database: 'tienda', port: 5432 // Descomenta y ajusta si es necesario
});

client.connect()
    .then(() => {
        console.log("Conexión exitosa");
        return client.end();
    })
    .catch(err => console.error(err));

File: conexiones/conexionPost.rb
require 'pg'

begin
  conn = PG.connect(
    host: "localhost",
    user: "postgres",
    password: "12345",
    dbname: "tienda"  # Ajusta el nombre de la base de datos según sea necesario
  )
  puts "Conexión exitosa a PostgreSQL"
  conn.close
rescue PG::Error => e
  puts "Error de conexión: #{e.message}"
end
